lib/src/models/data/schedule.dart

```diff
@@ -73,10 +72,22 @@ class ScheduleModel extends Model {
 					day == null ? null : Day.fromJson(day)
 			]
 		];
+
 		setToday();
 		notifyListeners();
 	}
 
+	///
+	Future<Map<String, Schedule>> getDefaultSchedules() async {
```

Comment:
```suggestion
	Future<void> setDefaultSchedules() async {
```

Since you're actually _setting_ the value of `Schedule.defaults` here, maybe rename this to `setDefaultSchedules`. `get_` implies you're returning a value you need to handle. Also, be sure to return `Future<void>` since you're not actually computing anything.